extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -460,28 +460,26 @@ private function isWithinRegistrationPerEmailLimit( $email ) {
 	}
 
 	/**
-	 * @desc Checks if the user/email is not blocked in phalanx and sets the proper response if not
+	 * @desc Checks if the user/email is blocked in phalanx and sets the proper response if so
 	 *
 	 * @param $user
 	 * @return bool
 	 */
-	private function isNotBlockedByPhalanx( $user ) {
+	private function isBlockedByPhalanx( $user ) {
+		$abortError = '';
 
-		return UserLoginHelper::callWithCaptchaDisabled(function($params) {
```

Comment:
This code runs fine without wrapping it in the callWithCaptchaDisabled method here so I just removed it.